tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -934,6 +934,10 @@ const reducer = createReducer(
       nextEndStep = nextStartStep;
     }
 
+    // When end step value is provided but useRangeSelectTime is still false,
```

Comment:
Perhaps clearer to avoid mentioning "flipping" (because this line doesn't necessarily flip anything). Something like: If there is no endStep then current selection state is single. Otherwise selection state is range.